packages/store/src/utils/selector-utils.ts

```diff
@@ -66,26 +82,57 @@ export function createSelector<T extends (...args: any[]) => any>(
     }
   };
 
-  selectorMetaData.selectFromAppState = fn;
+  selectorMetaData.selectFromAppState = selectFromAppState;
 
   return memoizedFn;
 }
 
+function setupSelectorMetadata<T extends (...args: any[]) => any>(
+  memoizedFn: T,
+  originalFn: T,
+  creationMetadata: CreationMetadata | undefined
+) {
+  const selectorMetaData = ensureSelectorMetadata(memoizedFn);
+  selectorMetaData.originalFn = originalFn;
+  let getExplicitSelectorOptions = () => ({});
```

Comment:
Why do you need this `getExplicitSelectorOptions` function? if it is never redefined

Reply:
It exists as a default value to use if the `creationMetadata` or the `creationMetadata.getSelectorOptions` is not provided.

Reply:
ok